config.sh

```diff
@@ -43,6 +43,8 @@ function run_tests_in_repo {
 		export PATH=$PWD:$PATH
 		# which pkg-config
 		# pip install matplotlib
+        yum install git
+        git --version
```

Comment:
Which code down the line needs git? In general I'd prefer to pip install from archives rather than cloning repos (if possible).